src/service/getIplayerExecutableService.ts

```diff
@@ -19,10 +20,13 @@ import socketService from './socketService';
 export class GetIplayerExecutableService {
     async getIPlayerExec(): Promise<GetIPlayerExecutable> {
         const fullExec: string = await configService.getParameter(IplayarrParameter.GET_IPLAYER_EXEC) as string;
-        const args: RegExpMatchArray = fullExec.match(/(?:[^\s"]+|"[^"]*")+/g) as RegExpMatchArray;
+        const args: RegExpMatchArray = fullExec?.match(/(?:[^\s"]+|"[^"]*")+/g) ?? ['get_iplayer'];
 
         const exec: string = args.shift() as string;
 
+        args.push('--encoding-console-out');
```

Comment:
Use UTF-8 on every platform, not just Mac/Linux: https://github.com/get-iplayer/get_iplayer/wiki/options#misc-options